SPRCLI/Sources/SPR/UpdateList.swift

```diff
@@ -63,9 +63,9 @@ extension SPRPublisher {
                 throw Error("URL is invalid")
             }
             return baseURL
-                .appending(component: scope)
-                .appending(component: name)
-                .appending(component: version)
+                .appendingPathComponent(scope)
+                .appendingPathComponent(name)
+                .appendingPathComponent(version)
```

Comment:
You will see several places where the method `appending(component:) is replaced by the older & now deprecated `.appendingPathComponent(_:)` so that these tools will compile on Swift 5.9.